average.js

```diff
@@ -1,6 +1,7 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  const newNumbers = numbers.filter(v => !Number.isNaN(v))
```

Comment:
Nice translation to concise code :)
We name variables so that readers can understand their intent without having to read all of the code.
Try another name instead of `newNumbers` to express its intent.